src/wwwroot/app.js

```diff
@@ -133,7 +160,30 @@
 
     }]);
 
-  dopplerRelayModule.run(['$rootScope', 'auth', '$location', '$translate', 'jwtHelper', function ($rootScope, auth, $location, $translate, jwtHelper) {
+  dopplerRelayModule.run([
+    '$rootScope', 
+    'auth', 
+    '$location', 
+    '$translate', 
+    'jwtHelper', 
+    '$locale',
+    function (
+      $rootScope,
+      auth,
+      $location, 
+      $translate, 
+      jwtHelper,
+      $locale) {
+
+    function applyCultureFormats() {
+      $locale.NUMBER_FORMATS.DECIMAL_SEP = $translate.instant("CULTURE_FORMATS.NUMBER_FORMATS.DECIMAL_SEP");
+      $locale.NUMBER_FORMATS.GROUP_SEP = $translate.instant("CULTURE_FORMATS.NUMBER_FORMATS.GROUP_SEP");
+    }
```

Comment:
> Also, we could take advantage of our translation variables convention and avoid `$translate.instant`, we could do something like:
> 
> ```js
> function applyCulture() {
>   var lang = $translate.use();
>   var cultureFormat = window['relay-translation-' + lang]['CULTURE_FORMAT'];
>   angular.merge($locale, cultureFormat);
> }
> ```
> 
> @mvillaseco what do you think about this?
> 
> I know that we are cheating using our global variable, but, it is better than remembering to change it each time that we add a new value.
> 
> @diegofox2  you will add new formats for dates, what do you think?
> 



I know that using the global variable is a smell, but it is an old smell hehe. Using _instant_ with generated keys, and making the merge manually is also a smell, right?

I applied my change in andresmoschini/relay-webapp@21c205b43ad6d5ede2810757467117b22808c3db it will be directly reusable for other formats like dates.

Could you take a look and merge in your branch if you like it?